addons/account_facturx/data/facturx_templates.xml

```diff
@@ -39,6 +40,12 @@
                                 <ram:CalculationPercent t-esc="line.discount"/>
                             </ram:AppliedTradeAllowanceCharge>
                         </ram:GrossPriceProductTradePrice>
+                        <!-- Line unit price, with discount applied -->
+                        <ram:NetPriceProductTradePrice>
+                            <ram:ChargeAmount
+                                t-att-currencyID="currency.name"
+                                t-esc="format_monetary(line.price_subtotal/line.quantity, currency)"/>
```

Comment:
Do we have something to handle the case "quantity = 0" ? (it sounds dumb but it happens that people invoice merchandise at a 0 based value)

Reply:
@Levizar @vin-odoo I have been hit by this case, and proposing the fix in #83776, and I think it's not so corner.